python/cuda_parallel/tests/test_reduce_api.py

```diff
@@ -16,11 +16,10 @@ def test_device_reduce():
     def op(a, b):
         return a if a < b else b
 
-    dtype = numpy.int32
-    h_init = numpy.array([42], dtype)
-    h_input = numpy.array([8, 6, 7, 5, 3, 0, 9])
-    d_output = cuda.device_array(1, dtype)
-    d_input = cuda.to_device(h_input)
+    dtype = np.int32
+    h_init = np.array([42], dtype=dtype)
```

Comment:
@gevtushenko do we care if the initial value lives on host or device?

Reply:
@leofang some API does support initial value in device memory, but most (reduce included) do not.

Reply:
Do we have an issue to track this disparity? Would be nice to create one. This may be confusing / not easy to teach.

Reply:
Tracked in https://github.com/NVIDIA/cccl/issues/2593.